lib/cpp/src/thrift/transport/TServerSocket.cpp

```diff
@@ -16,7 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+#define snprintf _snprintf
```

Comment:
This is already done in PlatformSocket.h - you can replace snprintf in this file with THRIFT_SNPRINTF and remove this code block instead.